neo.UnitTests/Persistence/UT_Store.cs

```diff
@@ -0,0 +1,182 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.IO.Caching;
+using Neo.IO.Wrappers;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.Persistence.LevelDB;
+using Neo.SmartContract.Manifest;
+using Neo.VM;
+using System;
+using System.IO;
+using System.Threading;
+
+namespace Neo.UnitTests.Persistence
+{
+    [TestClass]
+    public class UT_Store
+    {
+        private LevelDBStore store;
+        private string dbPath;
+
+        [TestInitialize]
+        public void TestSetup()
+        {
+            string threadName = Thread.CurrentThread.ManagedThreadId.ToString();
+            dbPath = Path.GetFullPath(nameof(UT_Store) + string.Format("_Chain_{0}", new Random().Next(1, 1000000).ToString("X8")) + threadName);
+            if (store == null)
+            {
+                store = new LevelDBStore(dbPath);
+            }
+        }
+
+        [TestCleanup]
+        public void DeleteDir()
+        {
+            store.Dispose();
+            store = null;
+            TestUtils.DeleteFiles(dbPath);
+        }
+
+        [TestMethod]
+        public void TestGetBlocks()
+        {
+            Snapshot snapshot = store.GetSnapshot();
+            TrimmedBlock block = new TrimmedBlock();
+            block.ConsensusData = new ConsensusData();
+            block.MerkleRoot = UInt256.Parse("0xa400ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff02");
+            block.PrevHash = UInt256.Parse("0xa400ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff01");
+            block.Timestamp = new DateTime(1968, 06, 01, 0, 0, 0, DateTimeKind.Utc).ToTimestamp();
```

Comment:
Hello,
This date is not "adequate". It is before the "big bang" (01/01/1970). Could you use a more "reliable" date? I don't know the effects of this usage. Probably an overflow?

Reply:
OK，I will update it. Perhaps there is a risk when the date is before 1970.